src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,55 @@
+.nav {
+  &__list {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    display: flex;
+  }
+
+  &__item {
+    margin-right: 20px;
+  }
+
+  &__item:last-child {
+    margin-right: 0;
+  }
```

Comment:
```suggestion
  &__item:not(:last-child) {
    margin-right: 20px;
  }
```

Reply:
I don't think it'll make any difference

Reply:
I don't think it makes any difference - it makes :)